slang-gfx.h

```diff
@@ -1314,6 +1399,18 @@ class IRenderCommandEncoder : public ICommandEncoder
     virtual SLANG_NO_THROW void SLANG_MCALL draw(UInt vertexCount, UInt startVertex = 0) = 0;
     virtual SLANG_NO_THROW void SLANG_MCALL
         drawIndexed(UInt indexCount, UInt startIndex = 0, UInt baseVertex = 0) = 0;
+    virtual SLANG_NO_THROW void SLANG_MCALL drawIndirect(
+        uint32_t maxDrawCount,
+        IBufferResource* argBuffer,
+        uint64_t argOffset,
```

Comment:
Minor question: should  there be a utility `BufferAndOffset` type to capture this idiom since it appears a few times?

Reply:
We can add it, but using it across the entire header would be a very pervasive change so I'd rather not do it now.